src/polls/PollForm.vue

```diff
@@ -12,7 +12,8 @@
           id="poll_title"
           placeholder="Заголовок"
           rows="3"
-          @input="v$.pollData.title.$touch"
+          @blur="v$.pollData.title.$touch"
+          @input="isDisabled = false"
```

Comment:
Will the save button be disabled if we'll type something here, and then delete?

Reply:
The button becomes disabled if you are editing the poll. If you are creating it. it remains active all the time whatever fields you fill/don't fill

Reply:
@aleonovdementyev  Should it be disabled in poll creation as well as in poll edition?

Reply:
updated: we have found common ground to leave it in way it works now